tests/fcidump/output.ref

```diff
@@ -1,1061 +1,24 @@
-     X:    -0.0000      Y:     0.0000      Z:    -0.0000     Total:     0.0000
-
-
-*** tstop() called on psinet at Mon Jul  2 16:11:51 2018
-Module time:
-	user time   =       0.09 seconds =       0.00 minutes
-	system time =       0.01 seconds =       0.00 minutes
-	total time  =          0 seconds =       0.00 minutes
-Total time:
-	user time   =       0.61 seconds =       0.01 minutes
-	system time =       0.05 seconds =       0.00 minutes
-	total time  =          1 seconds =       0.02 minutes
-Writing integrals in FCIDUMP format to Ne.C1.cc-pVDZ.UHF.INTDUMP
- MINTS: Wrapper to libmints.
-   by Justin Turney
-
-   Calculation information:
-      Number of threads:                 1
-      Number of atoms:                   1
-      Number of AO shells:               6
-      Number of SO shells:               6
-      Number of primitives:             22
-      Number of atomic orbitals:        15
-      Number of basis functions:        14
-
-      Number of irreps:                  1
-      Integral cutoff                 0.00e+00
-      Number of functions per irrep: [  14 ]
-
- OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
-         stored in file 35.
-
-      Computing two-electron integrals...done
-      Computed 861 non-zero two-electron integrals.
-        Stored in file 33.
-
-	Presorting SO-basis two-electron integrals.
-	Sorting File: SO Ints (nn|nn) nbuckets = 1
-	Transforming the one-electron integrals and constructing Fock matrices
-	Starting AA/AB first half-transformation.
-	Sorting AA/AB half-transformed integrals.
-	Starting BB first half-transformation.
-	Sorting BB half-transformed integrals.
-	First half integral transformation complete.
-	Starting AA second half-transformation.
-	Starting AB second half-transformation.
-	Starting BB second half-transformation.
-	Two-electron integral transformation complete.
-Integral transformation complete!
-Writing TEI integrals in FCIDUMP format to Ne.C1.cc-pVDZ.UHF.INTDUMP
-Writing frozen core operator in FCIDUMP format to Ne.C1.cc-pVDZ.UHF.INTDUMP
-Writing orbital energies in FCIDUMP format to Ne.C1.cc-pVDZ.UHF.INTDUMP
-Writing frozen core + nuclear repulsion energy in FCIDUMP format to Ne.C1.cc-pVDZ.UHF.INTDUMP
-Done generating Ne.C1.cc-pVDZ.UHF.INTDUMP with integrals in FCIDUMP format.
-	Ne, C1, cc-pVDZ, UHF reference, integrals in FCIDUMP format. 1-electron energyPASSED
-	Ne, C1, cc-pVDZ, UHF reference, integrals in FCIDUMP format. 2-electron energyPASSED
-	Ne, C1, cc-pVDZ, UHF reference, integrals in FCIDUMP format. SCF total energyPASSED
-	Ne, C1, cc-pVDZ, UHF reference, integrals in FCIDUMP format. MP2 correlation energyPASSED
-	Ne, C1, cc-pVDZ, UHF reference, integrals in FCIDUMP format.......PASSED
-
-    Psi4 stopped on: Monday, 02 July 2018 04:11PM
-    Psi4 wall time for execution: 0:00:01.14
-
-*** Psi4 exiting successfully. Buy a developer a beer!
+    FCIDUMP header........................................................................PASSED
```

Comment:
new isn't the same file as replacement. probably the output.ref doesn't need regenerating since the code changes in the PR are only to the fcidump file.

Reply:
Oops missed that one, you're right, I've just reverted output.ref